order/buyer/reissue.md

```diff
@@ -0,0 +1,49 @@
+---
+description: How to reissue an order as a buyer
+---
+
+# Reissue order
+
+As buyer you can send either a [new](issue.md) or updated purchase order to Tradecloud.
+
+## Send an updated order to Tradecloud
+
+As a buyer you can send an updated purchase order to Tradecloud.
+
+{% hint style="danger" %}
+Most supplier ERP integrations do not have the capability to automatically process an updated order. It will processed manually by the supplier and the order will have a longer human response time.
+{% endhint %}
+
+{% hint style="info" %}
+If an order line has order process status `Issued` or `In Progress` it will be `Reissued` and keep the same status.
+If the line has status `Rejected` (by supplier) it will be `Reissued` and become `In Progress`.
+If the line has status `Confirmed` it will be `Reopened` and become `In Progress`.
+In case of any other status like `Completed` or `Cancelled` the order update will be ignored.
+{% endhint %}
+
+### Send updated order API method
+
+The `/order-integration/order` API method is the same as when [sending a new order](issue.md) with additional JSON objects as mentioned below. Tradecloud will update the order based on the `purchaseOrderNumber` and will update or add lines based on the `lines.position` and will update or add delivery schedule and history based on `deliverySchedule.position` and `deliveryHistory.position`.
```

Comment:
Proposed:
"Tradecloud will update the order based on the `purchaseOrderNumber` and will add or update lines, delivery schedules and delivery histories where needed. Line matching will be done based on `lines.position`."

Reply:
Improved